src/main/pages/setup-kubernetes/kubernetes-config.md

```diff
@@ -66,4 +66,11 @@ When Eclipse Che gets deployed to Kubernetes, a PVC `che-data-volume` is [create
 * `kubectl describe pvc che-data-claim`, find PV it is bound to, then `oc describe pv $pvName`, you will get a local path with logs directory. Be careful with permissions for that directory, since once changed, Che server wont be able to write to a file
 * in Kubernetes web console, eclipse-che namespace, **pods > che-pod > logs**.
 
+## Che Workspace Termination Grace Period
+
+Grace termination period of Kubernetes / OpenShift workspace's pods defaults '0', which allows to terminate
```

Comment:
Please add a note that user-defined termination period will be used if it is specified.

Reply:
good point. lemme update

Reply:
Also please add the same information to [che.properties](https://github.com/eclipse/che/pull/9118/files#diff-2de0b0a873741f8e0d0879a46f8d5266R419) file.

Reply:
sure, that's the plan ;-)